src/CMakeLists.txt

```diff
@@ -314,11 +309,10 @@ if (USE_HIP)
   set_source_files_properties(${DBCSR_ACC_SRCS}
                               PROPERTIES HIP_SOURCE_PROPERTY_FORMAT 1)
   hip_add_library(acc STATIC ${DBCSR_ACC_SRCS})
-  target_include_directories(acc PRIVATE ${HIP_PATH}/../include)
   target_compile_definitions(acc PRIVATE ARCH_NUMBER=${ACC_ARCH_NUMBER})
   target_compile_definitions(acc PRIVATE __HIP)
-  target_compile_options(acc PRIVATE "-fPIC")
-  target_link_libraries(acc PUBLIC ${HIPBLAS})
+  set_property(TARGET acc PROPERTY POSITION_INDEPENDENT_CODE ON)
```

Comment:
```suggestion
  set_target_properties(acc PROPERTIES POSITION_INDEPENDENT_CODE ON)
```

Reply:
It was only necessary for this object library with timing functions. I've never seen an object lib before in cmake, but I've just left it there and added fPIC.